### DISCUSSION TITLE
Welcome to copilot Discussions!

### DISCUSSION THREAD

**Discussion by hcphoon01** [MultiCrew/copilot#161] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by calumshep** [comment]
That's really cool - what a great initiative! Can't wait to see what this brings...
